### PR TITLE
Bump tastypie to 0.9.12

### PR DESCRIPTION
**Pull request by jgrund**
0.9.11 only has options to do API key auth
with a querystring for GET.
0.9.12 has revamped auth code, and allows credentials to be sent via header as they should be.

Bump tastypie to 0.9.12.

### CODE REVIEW

File: django-tastypie/django-tastypie.spec [hunk 1]
```diff
@@ -3,8 +3,8 @@
 
 
 Name:           django-tastypie
-Version:        0.9.11
-Release:        5%{?dist}
+Version:        0.9.12
+Release:        0%{?dist}
```
Comment (brianjmurrell) [inline django-tastypie/django-tastypie.spec]: So normally Release: values start at 1, not 0.

I don't however, want our 0.9.12-1 to be considered equal to upstream providing such a release, and if they did provide such a release, our installations should upgrade to it, so I tend to make my release numbers the release I am updating+0.0[1-...].

So an appropriate value here I think would be `0.01{?dist}`.  I think that is just more obvious that we are just creeping into the next release without it looking like it was a mistake of using 0 instead of 1 as the initial release.

Comment (jgrund) [inline django-tastypie/django-tastypie.spec]: Works for me

File: test_inside_docker.sh [hunk 2]
```diff
@@ -26,8 +26,11 @@ EOF
 
 eval $(grep -e "^changed_dirs=" /manager-for-lustre-dependencies/env)
 
-useradd mocker
+groupadd --gid $(stat -c '%g' /manager-for-lustre-dependencies) mocker
+useradd --uid $(stat -c '%u' /manager-for-lustre-dependencies) --gid $(stat -c '%g' /manager-for-lustre-dependencies) mocker
 usermod -a -G mock mocker
+id
```
Comment (brianjmurrell) [inline test_inside_docker.sh]: Did you intend to leave this in?

File: test_inside_docker.sh [hunk 3]
```diff
@@ -26,8 +26,11 @@ EOF
 
 eval $(grep -e "^changed_dirs=" /manager-for-lustre-dependencies/env)
 
-useradd mocker
+groupadd --gid $(stat -c '%g' /manager-for-lustre-dependencies) mocker
+useradd --uid $(stat -c '%u' /manager-for-lustre-dependencies) --gid $(stat -c '%g' /manager-for-lustre-dependencies) mocker
```
Comment (brianjmurrell) [inline test_inside_docker.sh]: I wonder if this should be a separate commit since it's not really part of the update of django-tastypie. I wouldn't -1 for it but just something to think about.

File: test_inside_docker.sh (hunk 2 above)
Comment (jgrund) [inline test_inside_docker.sh]: whoops,removing

File: test_inside_docker.sh (hunk 3 above)
Comment (jgrund) [inline test_inside_docker.sh]: Yes, it probably should be.
